priority_alerts/views.py

```diff
@@ -0,0 +1,105 @@
+from priority_alerts import (
```

Comment:
local imports might be at the end of imports

Reply:
Adjusted.